packages/braid-design-system/lib/components/TextLink/TextLink.docs.tsx

```diff
@@ -53,6 +53,24 @@ const docs: ComponentDocs = {
           </Text>,
         ),
     },
+    {
+      label: 'Icons',
+      description: (
+        <Text>
+          For decoration or help distinguishing between links, an{' '}
+          <Strong>icon</Strong> can be provided. This will be placed to the left
+          of the link text.
+        </Text>
+      ),
+      Example: () =>
+        source(
+          <Text>
+            <TextLink icon={<IconLink />} href="">
```

Comment:
Is it worth adding a snippet with this pattern?

Reply:
Maybe, not sure how consistently the icon slots have been added to the snippets.

Given that is a site update, lets cut this release and we can double back for icon slot snippets without needing to release.